### PR TITLE
Don't use conda-execute

### PR DESCRIPTION
**Pull request by isuruf**
Temporary workaround until https://github.com/conda-tools/conda-execute/issues/46 is fixed

### CODE REVIEW

**Comment by jakirkham** [comment]
Should we do the same for the feedstocks repo?

**Comment by isuruf** [comment]
Sure, but I want to remove the `update_teams.py` part from that after https://github.com/conda-forge/staged-recipes/pull/3975

**Comment by jakirkham** [comment]
Completely agreed. Also want to get the feedstocks stuff out once we have that handled too.

File: .ci_scripts/update_feedstocks_repo [hunk 1]
```diff
@@ -15,6 +16,6 @@ if [ -n "$GH_TOKEN" ]; then
     cd ../
 
     echo "Updating the teams"
-    conda execute -v ./scripts/update_teams.py ./feedstocks_repo/feedstocks
+    pytho scripts/update_teams.py ./feedstocks_repo/feedstocks
```
Comment (jakirkham) [inline .ci_scripts/update_feedstocks_repo]: Should be `python`.

Comment (isuruf) [reply .ci_scripts/update_feedstocks_repo]: good catch

**Comment by jakirkham** [comment]
Thanks!

**Comment by isuruf** [comment]
Docs are up, https://conda-forge.org/docs/meta.html#building-noarch-packages

**Pull request merged**